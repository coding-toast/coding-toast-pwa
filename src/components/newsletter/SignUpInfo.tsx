import * as React from 'react';

const SignUpInfo: React.FC = () => {
  return (
    <div>
      <h1>Follow us</h1>
    </div>
  );
};

export default SignUpInfo;
