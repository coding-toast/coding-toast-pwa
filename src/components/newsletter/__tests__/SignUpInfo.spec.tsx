import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SignUpInfo from '../SignUpInfo';

describe('<SignUpInfo />', () => {
  it('renders correctly', () => {
    const component = shallow(<SignUpInfo />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
