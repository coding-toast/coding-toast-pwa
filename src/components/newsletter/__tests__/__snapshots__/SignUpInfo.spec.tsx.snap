// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUpInfo /> renders correctly 1`] = `
<div>
  <h1>
    Follow us
  </h1>
</div>
`;
